fortran/src/hdf5_fortrandll.def.in

```diff
@@ -160,21 +193,29 @@ H5I_mp_H5IIS_VALID_F
 ; H5L
 H5L_mp_H5LCOPY_F
 H5L_mp_H5LDELETE_F
+H5L_mp_H5LDELETE_ASYNC_F
 H5L_mp_H5LCREATE_SOFT_F
+H5L_mp_H5LCREATE_SOFT_ASYNC_F
 H5L_mp_H5LCREATE_HARD_F
+H5L_mp_H5LCREATE_HARD_ASYNC_F
 H5L_mp_H5LCREATE_EXTERNAL_F
 H5L_mp_H5LDELETE_BY_IDX_F
+H5L_mp_H5LDELETE_BY_IDX_ASYNC_F
 H5L_mp_H5LEXISTS_F
+H5L_mp_H5LEXISTS_ASYNC_F
 H5L_mp_H5LGET_INFO_F
 H5L_mp_H5LGET_INFO_BY_IDX_F
 H5L_mp_H5LIS_REGISTERED_F
 H5L_mp_H5LMOVE_F
 H5L_mp_H5LGET_NAME_BY_IDX_F
 H5L_mp_H5LITERATE_F
+H5L_mp_H5LITERATE__ASYNC_F
```

Comment:
Is this meant to have two underscores in the name?

Reply:
thanks, fixed.